TarSCM/scm/base.py

```diff
@@ -147,7 +147,9 @@ def prepare_clone_dir(self):
         # current work directory to allow the developer to work inside
         # of the git repo and fetch local changes
         if sys.argv[0].endswith("snapcraft") or \
-           (self.args.use_obs_scm and os.getenv('OSC_VERSION')):
+           (self.args.use_obs_scm and
+                os.getenv('OSC_VERSION') and
+                os.path.isdir('.git')):
```

Comment:
LGTM.  I'm wondering if this fixes #202 - even if not it looks related at first glance.